api/agent/drivers/docker/docker.go

```diff
@@ -203,6 +210,25 @@ func (drv *DockerDriver) Prepare(ctx context.Context, task drivers.ContainerTask
 	return &cookie{id: task.Id(), task: task, drv: drv}, nil
 }
 
+func (drv *DockerDriver) SupportsLimit(ctx context.Context, l drivers.Limit) bool {
```

Comment:
We can improve later, but just noting it here: we do fetch docker info in checkDockerVersion() in docker.go. We should probably cache the results for referencing it below.